examples/python-dataflow/run.rs

```diff
@@ -1,4 +1,4 @@
-use dora_core::{get_pip_path, get_python_path, run};
+use dora_core::{get_python_path, run};
```

Comment:
So, in certain ubuntu `pip` might refer to `pip2` and `pip3` might refer to the actual pip3, therefore it would be better to use `get_pip_path`

Reply:
I don't think we have to get pip path, the run.rs script only uses python to generate an virtual env
```Rust 
run(
        get_python_path().context("Could not get python binary")?,
        &["-m", "venv", "../.env"],
        None,
    )
```

Reply:
Don't we use `pip` to install maturin?

Reply:
Oh yeah i just understood what i was doing wrong